### PR TITLE
Code review2 haseebikram

### PR DESCRIPTION
**Pull request by haseebikram**
Analysis of the program:
This program allows you to take a photo yourself or it takes a random photo for you from the API to be able to use it as a puzzle and solve it.

Was the program available in UC Github on time?:
Yes, the program was available in UC Github.

Is the program documented/commented well enough for you to understand?:
Yes, the program was documented and commented well for us to understand.

Does the program compile?:
Yes, the program compiles.

Three skills I learned from this code review:
- The code letters and numbers for the color pallet
- How to change the names of colors & minimizing it
- Button styles (borderless, colored, etc)

Links to three specific commits that I made to my own group's GitHub repository after Sprint 1's deadline, and before Sprint 2's deadline: (Pool Test)
92f4b11
1f32dd5
a0313b0

### CODE REVIEW

**Comment by haseebikram** [comment]
Comment for commit #1 (Change color name to cream)

I changed the name to "cream" from  "creamBackground" because it allowed for a simpler id when naming the color, making the color code cleaner & more minimal.

Source:
https://kotlinlang.org/docs/reference/coding-conventions.html

**Comment by haseebikram** [comment]
Comment for commit #2 (Updating black color)

I changed the code for the black to #000000 from #00000000, which allows it to be shorter but the same black color they wanted, allowing cleaner code and minimal numbers.

Source: 
https://kotlinlang.org/docs/reference/coding-conventions.html

**Comment by haseebikram** [comment]
Comment for commit #3 (Updating button)

I changed the button from borderless, to be colored. Therefore, the button is much more distinguished and stands out more as it matches the accent colors.

Source:
https://kotlinlang.org/docs/reference/coding-conventions.html

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <color name="white">#FFFFFFFF</color>
 
     // custom color theme
-    <color name="creamBackground">#BFB2A3</color>
+    <color name="cream">#BFB2A3</color>
```
Comment (discospiff) [inline app/src/main/res/values/colors.xml:12]: Was this changed where used as well?

**Comment by discospiff** [comment]
1) I don't recommend merging this branch, as it does not reduce technical debt.

2) @haseebikram  merged it, and should not have.  Please revert this merge.  The group should decide whether or not to merge, not the reviewer.

**Pull request merged**